macros/schema_tests/column_values_basic/expect_column_values_to_be_decreasing.sql

```diff
@@ -22,7 +28,12 @@ add_lag_values as (
     select
         sort_column,
         value_field,
-        lag(value_field) over(order by sort_column) as value_field_lag
+        lag(value_field) over
+            {%- if not partition_columns -%}
+                (order by sort_column)
+            {%- else -%}
+                (partition by {{ partition_columns }} order by sort_column)
```

Comment:
Since `partition_columns` is a list, you'll need to either loop, or do a `.join(",")` (easier) here.